spec/system/admin/products_v3/products_spec.rb

```diff
@@ -1419,23 +1419,22 @@
 
     it "shows only suppliers that I manage or have permission to" do
       visit spree.admin_products_path
-
       within row_containing_name(product_supplied.name) do
-        page.has_select?(
-          '_products_0_supplier_id',
-          options: [
-            supplier_managed1.name, supplier_managed2.name, supplier_permitted.name
-          ], selected: supplier_managed1.name
-        )
+        expect(page.has_select?(
```

Comment:
This is fine, but FYI you can use the equivalent form:

`expect(page).to have_select(`

(RSpec magically handles a `have_*` matcher for any `has_*?` methods)

Reply:
Thanks @dacook , it was working but the `have_select()` syntax is really much simpler. Squashed it to the previous commit.